scss/foundation-styles/mixins.scss

```diff
@@ -88,3 +88,30 @@
     }
   }
 }
+
+@mixin homepage-search-dropdown( $border-color, $dropdown ) {
```

Comment:
I created this mixin to differentiate between statement form elements on the homepage and regular ones throughout the site.